app/views/components/_topic-list.html.erb

```diff
@@ -1,20 +1,22 @@
 <%
   items ||= []
   see_more_link ||= false
+  size ||= false
+  component_size = "app-c-topic-list--small" if size == 'small'
```

Comment:
Would passing in `small: true` make more sense here? Otherwise people might make typos and/or expect other sizes to be able to be passed in.

Reply:
Ha. I came up with both options but thought if I used small: true someone would say why not do a general one for size? But I guess we don't build things that we don't need yet, so I'll take a look.

On the fixup, I think I tried but it didn't work. I'll give it another attempt - I might be thinking of another PR.

Reply:
@vanitabarrett I can't fixup that commit - I must have changed something that means the commits can't be reordered. Have addressed the other point though.